### DISCUSSION TITLE
I don't understand what the collision detection is doing.

### DISCUSSION THREAD

**Discussion by jimlynch22** [ZL-Audio/ZLEqualizer#83] [Q&A] [answered]
I don't understand what the collision detection is doing. The manual says it compares the main-chain signal and the side-chain signal. In my testing, I had one track and no sidechain, but it still detects collisions. what gives?

**Comment by zsliu98** [comment, marked as answer]
If you do not press the S button, the side-chain is from the internal main-chain, i.e., you are detecting the collisions between two original main-chain inputs (when lookahead is zero). It may represent the peaks on the spectrum.

**Reply by zsliu98** [reply]
I'd better add a footnote to the manual since someone also asked this question a while ago under my YouTube video :grin:

**Reply by jimlynch22** [reply]
I still don't understand.  I tried setting latency=0 and S button is off. I play a single track and collisions are still detected.  By the way, the manual indicates a Zero Latency on/off button, but I don't have that.

![Screenshot 2024-07-15 085127](https://github.com/user-attachments/assets/ee8264f9-03f7-49e6-bdb1-ecbde72ec573)
![Screenshot 2024-07-15 085423](https://github.com/user-attachments/assets/d8527193-d713-4748-ba40-be1da788128b)

**Reply by zsliu98** [reply]
@jimlynch22  As I said, if you do not press the S button, you are detecting collisions between the signal and itself. Generally, you should not use collision detection in this way.
The manual is up-to-date with the latest version, i.e., 0.3.3 Pre-Release. See https://github.com/ZL-Audio/ZLEqualizer/releases. I have just released it yesterday. If you don't need zero latency or AAX, I would suggest against upgrading.

**Reply by jimlynch22** [reply]
ok thanks.
